stdlib/public/SDK/Foundation/Data.swift

```diff
@@ -98,47 +102,140 @@ public final class _DataStorage {
     public var _needToZero: Bool
     public var _deallocator: ((UnsafeMutableRawPointer, Int) -> Void)?
     public var _backing: Backing = .swift
+    public var _offset: Int
     
     public var bytes: UnsafeRawPointer? {
         @inline(__always)
         get {
             switch _backing {
             case .swift:
```

Comment:
Don't know how much this matters, but you should be able to shorten this a little bit:

```swift
let bytesPtr: UnsafeRawPointer?
switch _backing {
case .swift: fallthrough
case .immutable: fallthrough
case .mutable: bytesPtr = _bytes

case .customReference(let d): bytesPtr = d.bytes
case .customMutableReference(let d): bytesPtr = d.bytes
}

return bytesPtr?.advanced(by: -_offset)
```